### PR TITLE
Big Product Slideshow

### PR DESCRIPTION
**Pull request by atironati**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,6 +5,35 @@
...
+            breakpoints: {
+                760: {
+                  perView: 2
+                },
+                480: {
+                    perView: 1
+                }
+            }
+        });
+
```
Comment (ColinRTaylor) [inline index.html:33]: I really think this code and both company's customers could really benefit from abstracting this furthur. I recommend: 
```javascript
function makeMagicFountain() {
  
  var popcornPie = {
   dominion: products.mount,
    serendipity() {
      return this.dominion;
    },
    acquiescence() {
      this.serendipity()();
    },
  };
  
  return Object.create(obj).acquiescence()
}
```

File: index.html [hunk 2]
```diff
@@ -24,29 +53,49 @@
       Welcome to Both Company, where dreams become reality. Our stunning lineup of quality products is nothing short of GENIUS
     </div>
 
-    <div class="product-container">
-      <div class="product">
-        <h2 class="product-title" >Napkin Rings!</h3>
+    <div class="glide product-container">
+      <div data-glide-el="track" class="glide__track">
+        <ul class="glide__slides">
```
Comment (ColinRTaylor) [inline index.html:72]: let's rename this class to `whiteTailDove`

File: index.html [hunk 3]
```diff
@@ -24,29 +53,49 @@
...
+          <li class="glide__slide">
```
Comment (ColinRTaylor) [inline index.html:73]: This should be called `ty$siGn`

**Review by ColinRTaylor — CHANGES_REQUESTED** [review]
how wonderful

File: index.html (hunk 1 above)
Comment (atironati) [reply index.html:33]: I integrated in this code but it didn't actually do anything so not really sure what you were intending here lol

File: index.html (hunk 2 above)
Comment (atironati) [reply index.html:72]: done

File: index.html (hunk 3 above)
Comment (atironati) [reply index.html:73]: the $ character was messing with things so I took some creative liberty and renamed this to tyDollaSlide I hope you approve

**Pull request merged**